### PR TITLE
Support Solidity 0.8.19

### PR DESCRIPTION
**Pull request by blitz-1306**
## Changes
- [x] Updated dependencies.
- [x] Updated affected test artifacts. Add test sample to use operators for custom types.
- [x] Tweak logic to handle tuple types with empty elements.
- [x] Tweak call graph to respect operators for custom types.

Regards.

### CODE REVIEW

File: src/instrumenter/callgraph.ts [hunk 1]
```diff
@@ -33,6 +35,25 @@ export interface CallGraph {
     overridenBy: FunMap;
 }
 
+function findCallersAndCallees(fun: FunctionDefinition, callers: FunMap, callees: FunMap): void {
```
Comment (cd1m0) [inline src/instrumenter/callgraph.ts:38]: Can we rename this function to `processCallNodes` ?

**Review by cd1m0 — CHANGES_REQUESTED** [review]
LGTM. I need to do a pass and add TC logic to support custom operators in annotations

File: src/instrumenter/callgraph.ts (hunk 1 above)
Comment (blitz-1306) [reply src/instrumenter/callgraph.ts:38]: Sure, will do.

**Review by cd1m0 — APPROVED** [review]
LGTM

**Pull request merged**